src/css/base/base.css

```diff
@@ -111,3 +111,80 @@ main:has(.back-link) {
     height: 20px;
   }
 }
+
+/* #region Logo */
+/* Logo, reusable part */
+.logo {
+  color: var(--green-color);
+  font-weight: 700;
+  font-size: 20px;
+}
+/* #endregion */
+
+/* #region Nav links */
+/* Navigation links, reusable part */
+.nav_link {
```

Comment:
if it is main menu.
We need to styling in header.css

Reply:
The same links will be reused in footer, based on design from figma, the same text appearance, same hover, the only difference between links in header and footer is text color

Reply:
Ok